test/e2e/ha/core_contrail_services_test.go

```diff
@@ -337,9 +336,9 @@ func untaintNodes(k kubernetes.Interface, nodeLabelSelector string) error {
 }
 
 func assertReplicasReady(t *testing.T, w wait.Wait, r int32) {
-	t.Run(fmt.Sprintf("then a Zookeeper StatefulSet has %d ready replicas", r), func(t *testing.T) {
+	t.Run("then a Zookeeper StatefulSet has 1 ready replicas", func(t *testing.T) {
```

Comment:
please write a todo that it should be changed after fixing a bug with Zookeeper

Reply:
added